### ISSUE TITLE
straight LV cable is invisible under catwalk

### ISSUE THREAD

**Issue by lzk228** [space-wizards/space-station-14#32731]
![image](https://github.com/user-attachments/assets/df0b95fb-1793-4960-bd22-390689f743fd)

**Comment by AlexUm418** [comment]
Yeah so thats why I never saw lvs under these things. I thought it was my vision getting worse

**Comment by Moomoobeef** [comment]
I'll just move the sprites up by one pixel

**Comment by Moomoobeef** [comment]
![image](https://github.com/user-attachments/assets/3fde4f15-a884-463c-9b48-74d30f349a4f)
actually I can't go up or the 4-point corners won't be symmetrical...

I could go down by one pixel, which puts it closer to the MV but it's still not overlapping completely and will still fix the base issue.

**Comment by Moomoobeef** [comment]
![image](https://github.com/user-attachments/assets/45d2953f-2904-41d2-8ca5-ee160faa50da)
well fuck.

I'm either going to move LV to be on the opposite side of HV, or I'm going to move both HV and MV further down to make room for moving LV a further pixel down.
Please vote which one you think would look better before I do the work:
:+1: for moving LV to the other side of HV
:-1: for moving all cables down

**Comment by Moomoobeef** [comment]
![image](https://github.com/user-attachments/assets/e302e0a3-7edc-433f-88a0-b51468a2d7cc)
I ended up doing something different from both proposed solutions. I did not move HV, only moved MV and LV.
This does mean that MV is overlapping HV, but HV is wide enough that you can still clearly see it so I don't think it's a big deal, plus the further off-center I move HV the weirder the cable terminal will look when connected to things.

*speaking of the cable terminal*
![image](https://github.com/user-attachments/assets/74f1f50e-cd5f-43e0-bfa5-3404348796e1)
whenever the cables were made staggered in the first place the terminal wasn't adjusted. So I'll do that too, though it may or may not be in a separate PR
